ceph/ceph_admin/osd.py

```diff
@@ -106,3 +106,97 @@ def apply(self, config: Dict) -> None:
             sleep(interval)
 
         raise OSDServiceFailure("OSDs are not up and running in hosts")
+
+    def rm_status(self, config):
+        """
+        Execute the command ceph orch osd rm status.
+
+        Args:
+            config: command remove is passed from the test case.
```

Comment:
please add type hinting like mentioned in this code.
https://github.com/red-hat-storage/cephci/blob/master/ceph/ceph_admin/daemon.py#L13-L31

```
Args:
       config (Dict): OSD removal configuration 
```
same in case of `Returns` docstring section.